instrumentation/opentelemetry-api/opentelemetry-api-1.38/javaagent/src/noopTest/java/io/opentelemetry/javaagent/instrumentation/opentelemetryapi/v1_38/incubator/metrics/NoopTest.java

```diff
@@ -29,15 +30,19 @@ void noopInstance() {
     Meter meter = testing.getOpenTelemetry().getMeter("test");
 
     LongCounterBuilder counterBuilder = meter.counterBuilder("test");
-    assertThat(counterBuilder).isNotInstanceOf(ExtendedLongCounterBuilder.class);
+    assertThat(counterBuilder).isInstanceOf(ExtendedLongCounterBuilder.class);
+    ((ExtendedLongCounterBuilder) counterBuilder).setAttributesAdvice(emptyList());
```

Comment:
what is the purpose of this setting empty attributesAdvice?

Reply:
This test was introduced in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11934 We verified that when the underlying instrument is noop then our bridge did not implement the incubating api, because the noop instrument itself didn't implement it making bridging impossible. Now noop instruments implement the incubating api and so does our bridge, we call `setAttributesAdvice` as was done in https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/11928 to verify that this works and there are no exceptions. The argument list doesn't really matter for this test and using empty list should be fine.